app/src/main/java/com/iic/shopingo/api/Request.java

```diff
@@ -10,45 +19,56 @@
  */
 public class Request {
   public static OutgoingRequestAPIResult makeRequest(List<String> shoppingItems, int offer, String shopperId) {
```

Comment:
That's kinda weird that the request itself make the post, I would expect the server or API class to have a method to submit this request.